Source/SourceTerms/ERF_make_sources.cpp

```diff
@@ -60,8 +60,8 @@ void make_sources (int level,
     const bool use_terrain      = solverChoice.use_terrain;
 
     TurbChoice tc = solverChoice.turbChoice[level];
-    const bool l_use_deardorff  = (tc.les_type == LESType::Deardorff);
-    const bool l_use_QKE        = tc.use_QKE && tc.diffuse_QKE_3D;
```

Comment:
Looks like we lost the option toggle the 6th-order numerical diffusion term for KE/QKE alone. I think this was an experimental option and, if I recall correctly, didn't make much difference as far as numerical stability of MYNN was concerned.

Reply:
Let me put that back in just for consistency. I'd like to change as few things as possible with this PR.